test/new-e2e/agent-subcommands/configcheck_test.go

```diff
@@ -104,7 +104,7 @@ Config for instance ID: cpu:e331d61ed1323219
 
 // cpu, disk, file_handle, io, load, memory, network, ntp, uptime
 func (v *agentConfigCheckSuite) TestDefaultInstalledChecks() {
-	v.UpdateEnv(e2e.AgentStackDef(nil))
+	v.UpdateEnv(e2e.AgentStackDef(nil, nil))
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏ Why `nil, nil` ? Can't we just remove all arguments ?

Reply:
Thank you, removed !